content/docs/user-guide/index.md

```diff
@@ -1,51 +1,81 @@
-# Overview
+# What is DVC?
+
+<!--
+## What is DVC?
+-->
+
+**Data Version Control** is a [free], open-source tool for [data management],
+[ML pipeline][ml pipelines] automation, and [experiment management]. This helps
+data science and machine learning teams manage **large datasets**, make projects
+**reproducible**, and **collaborate** better.
+
+DVC takes advantage of the existing software engineering toolset your team
+already knows (Git, your IDE, CI/CD, etc.). Its design follows this set of
+principles:
+
+1. **Codification**: Define any aspect of your ML project (data and model
+   versions, ML pipelines and experiments) in human-readable [metafiles]. This
+   enables using best practices and established engineering toolsets, reducing
+   the gap with data science.
+1. **Versioning**: Use Git (or any SCM) to version and share your entire ML
+   project including its source code and configuration, parameters and metrics,
+   as well as data assets and processes by committing DVC metafiles (as
+   placeholders).
+1. **Secure collaboration**: Control the access to all aspects of your project
+   and share them with the people and teams you choose.
+
+[data management]: /doc/user-guide/data-management
+[manage data]: /doc/user-guide/data-management
+[ml pipelines]: /doc/user-guide/pipelines
+[experiment management]: /doc/user-guide/experiment-management
+[metafiles]: /doc/user-guide/project-structure
+
+## Characteristics
+
+- DVC comes as a [VS Code Extension], as a [command line] interface, and as a
+  [Python API]. These options provide a familiar an intuitive **user
+  experience** to a broad range of users.
+- **Easy to use**: DVC is quick to [install](/doc/install) and works out of the
+  box. It doesn't require special infrastructure, nor does it depend on APIs or
+  external services.
+
+  <admon type="tip">
+
+  Optional integrations with existing solutions and platforms such as Git
+  hosting, SSH and cloud storage providers, among others are included.
 
-## Core Features
+  </admon>
+
+- **Works on top of Git repositories** and has similar feel and flows. Stick to
+  the regular Git workflow (commits, branching, pull requests, etc.) and don't
+  reinvent the wheel!
 
-- DVC is a [free], open-source [VS Code Extension] and [command line] tool.
+  <admon type="info">
 
-- DVC works **on top of Git repositories** and has a similar command line
-  interface and flow as Git. DVC can also work stand-alone, but without
+  DVC can also work stand-alone, but without
   [versioning](/doc/use-cases/versioning-data-and-models) capabilities.
 
+  </admon>
+
+<!--
 - DVC codifies data and ML experiments:
 
   ![](/img/reproducibility.png)
+-->
```

Comment:
p.s. should we just kill this one? It's nice but I'm not sure where to put it.

![image](https://user-images.githubusercontent.com/1477535/213358775-8ee15c5c-ef61-4eb7-94b1-556ec87c6b95.png)